src/views/VallisFish.vue

```diff
@@ -2,171 +2,141 @@
   <b-container fluid>
     <b-col md="12">
       <router-link to="/vallis/map">
-        <b-button variant="info" class="btn-block mb-3">Orb Vallis Map</b-button>
+        <b-btn variant="info" class="btn-block mb-3">Orb Vallis Map</b-btn>
       </router-link>
-
+      <p style="color:firebrick">
+        <b>
+          All servofish in the Orb Vallis should be acquired with a shock spear to avoid damage to them
+        </b>
+      </p>
       <div class="row">
-        <b-table striped hover :items="items" class="fish-info b-table"></b-table>
+        <table class="table fish-info striped hover">
```

Comment:
any particular reason for using this instead of keeping to the model and letting it generate the table for us?

Reply:
I personally think the rowspan looks better, otherwise you'll either have a hacky solution to do rowspan within the dynamic table, or you'll have to live with the fact that you're going to have 3 rows with the same/empty information.

As of I know, b-table does not support rowspan.

Reply:
this 
![image](https://user-images.githubusercontent.com/7128721/60851771-bee64480-a1c2-11e9-852c-bc0ec8a2a2e9.png)

just feels more compact and usable than

![image](https://user-images.githubusercontent.com/7128721/60851792-d291ab00-a1c2-11e9-86f9-541937c38cfb.png)


the sizing on the links also feels wrong.

Reply:
I get the compactness, but do you want to sacrifice the images for the information? If so, I can revert it back and just fix the information.

Reply:
i'd be fine if just the links were fixed.

Reply:
I have made some edits to the page to retain its original compactness while keeping the pictures. Let me know if this is the direction you want to head in. I also don't want my opinion to overrule what you had going with b-tables. I do think that it's superior to have them in b-tables, as I don't have a nice solution to make it sortable without making them into datatables.